<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="image_perspective_transformer" pkg="image_perspective_transformer" type="image_perspective_transformer_node" output="screen" >
    <remap from="image_perspective_transformer/input_image" to="/usb_cam/image_raw" />
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="image_perspective_transformer/out_image" />
    <param name="autosize" value="true" />
  </node>
</launch>
